libs/libmbim/Makefile

```diff
@@ -0,0 +1,89 @@
+#
+# Copyright (C) 2016 Velocloud Inc.
+# Copyright (C) 2016 Aleksander Morgado <aleksander@aleksander.es>
+#
+# This is free software, licensed under the GNU General Public License v2.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libmbim
+PKG_VERSION:=1.18.2
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:=https://www.freedesktop.org/software/libmbim
+PKG_HASH=7723f2ca2286c5286ffba7331cb4520099982e55355a7af6f45bdce6b5d39268
+
+PKG_MAINTAINER:=Nicholas Smith <nicholas.smith@telcoantennas.com.au>
+
+PKG_INSTALL:=1
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/nls.mk
```

Comment:
Is nls needed?

Reply:
I thought that was needed for internationalisation, is it?

Reply:
@aleksander0m do you know?

Reply:
I don't think it's needed; libmbim does UTF-8 <-> UTF-16 conversions through glib and I believe I changed that long ago to avod any need of iconv()